### PR TITLE
Working BioPage with basic Calendar

### PR DESCRIPTION
**Pull request by DChan0319**

### CODE REVIEW

File: client/src/actions/index.js [hunk 1]
```diff
@@ -26,18 +29,46 @@ export const logOutUser = () => {
...
+  }
+
+  const body = {
+    pictureUrl: `http://s3-us-west-1.amazonaws.com/${process.env.S3_BUCKET}/${picture.filename}`,
+    userId: payload.user.id
+  };
+
   try{
-    await axios.post('https://api.cloudinary.com/v1_1/dxfzmbtst/image/upload', formData, {
+    const signedUrl = await axios.post('http://localhost:8000/api/aws/s3',picture);
```
Comment (zgallagher08) [inline client/src/actions/index.js]: add cookie to headers

File: client/src/actions/index.js [hunk 2]
```diff
@@ -26,18 +29,46 @@ export const logOutUser = () => {
...
+
+    await axios.put(signedUrl.data, payload.file[0], options);
```
Comment (zgallagher08) [inline client/src/actions/index.js]: add cookie to header

File: client/src/actions/index.js [hunk 3]
```diff
@@ -122,6 +153,32 @@ export const updateWorkoutsWithStar = (workouts) => {
   }
 };
 
+export const getUserPublicWorkoutsList = async(userId) =>{
+  try{
+    const publicWorkouts = await axios.get(`http://localhost:8000/api/workouts/public/user/${userId}`);
+    //console.log("public workouts********",publicWorkouts);
+    if(Array.isArray(publicWorkouts.data)){
+      for(let publicWorkout of publicWorkouts.data){
+        let exercises = await axios.get(`http://localhost:8000/api/workouts/exercises/${publicWorkout.id}`);
```
Comment (amelendez37) [inline client/src/actions/index.js]: add cookie to header

**Review by amelendez37 — APPROVED** [review]

File: client/src/components/workoutsView/workoutItem.jsx [hunk 4]
```diff
@@ -30,9 +30,12 @@ class WorkoutItem extends Component {
   }
 
   render() {
-    return (
+    return ( 
       <div className="workout-container">
-        {this.props.clickedWorkout ?
+      {!this.props.clickedWorkout ? null :
```
Comment (zgallagher08) [inline client/src/components/workoutsView/workoutItem.jsx:35]: copy/paste this file from github, no need to fix.

File: client/src/containers/workoutItemContainer.jsx [hunk 5]
```diff
@@ -49,10 +49,10 @@ class WorkoutItemContainer extends Component {
...
     return (
       <WorkoutItem
-         user_id={this.props.user_id}
-         clickedWorkout={this.props.clickedWorkout}
-         handleStarWorkoutClick={this.handleStarWorkoutClick}
-         star={star}
+       user_id={this.props.user_id}
+       clickedWorkout={this.props.clickedWorkout}
+       handleStarWorkoutClick={this.handleStarWorkoutClick}
+       star={!this.props.clickedWorkout? false : !!this.props.clickedWorkout.star}
```
Comment (zgallagher08) [inline client/src/containers/workoutItemContainer.jsx]: just change this to star={star}

**Review by zgallagher08 — CHANGES_REQUESTED** [review]
see comments

**Comment by DChan0319** [comment]
workoutItem.jsx doesn't work with current master version for the  hitting the BioPage. Works with my version though.

**Review by zgallagher08 — APPROVED** [review]

**Pull request merged**